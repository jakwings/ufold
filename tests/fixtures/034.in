米田共
SHIT
