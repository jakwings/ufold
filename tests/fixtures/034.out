米
田
共
S
H
I
T
